common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -2444,6 +2447,24 @@ object CelebornConf extends Logging {
       .stringConf
       .createWithDefault("/sys/block")
 
+  val WORKER_DISK_MONITOR_NOTIFY_ERROR_THRESHOLD: ConfigEntry[Int] =
+    buildConf("celeborn.worker.monitor.disk.notifyError.threshold")
+      .categories("worker")
+      .version("0.2.0")
+      .doc("Device monitor will only notify critical error once the accumulated valid non-critical error number " +
+        "exceeding this threshold.")
+      .intConf
+      .createWithDefault(64)
+
+  val WORKER_DISK_MONITOR_NOTIFY_ERROR_EXPIRE_TIMEOUT: ConfigEntry[Long] =
+    buildConf("celeborn.worker.monitor.disk.notifyError.expireTimeout")
+      .categories("worker")
+      .version("0.2.0")
+      .doc("The expire timeout of non-critical device error. Only notify critical error when the number of non-critical " +
+        "errors for a period of time exceeds threshold.")
+      .timeConf(TimeUnit.MILLISECONDS)
+      .createWithDefaultString("1h")
```

Comment:
Should ensure the timeout is not much greater than the device monitor `diskCheckInterval`, because currently iohang is considered as non-critical error, we should ensure if iohang happened it can trigger the critical error in time. BTW, consider to isolate stuck working dir when iohang happens rather than notifyNonCriticalError.